.github/workflows/c1-v2.yml

```
@@ -0,0 +1,186 @@
+          docker build -f deploy/server/Dockerfile-v2 -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+          docker build -f deploy/server/Dockerfile-v2 -t $ECR_REGISTRY/$ECR_REPOSITORY:latest .
+
+      - name: Push V2 image to ECR
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: lifebookshelf-server-v2
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)